Makefile

```diff
@@ -324,6 +348,11 @@ golint: get-ci-tools
 	GOWORK=off PATH=$(GOBIN):$(PATH); $(CI_TOOLS_REPO_DIR)/test-runner/golint.sh
 	GOWORK=off PATH=$(GOBIN):$(PATH); $(CI_TOOLS_REPO_DIR)/test-runner/golint.sh ./api
 
+.PHONY: operator-sdk
+operator-sdk: $(OPERATOR_SDK) ## Download operator-sdk locally if necessary.
+$(OPERATOR_SDK): $(LOCALBIN)
+	test -s $(OPERATOR_SDK) || curl -o $(LOCALBIN)/operator-sdk -L https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_linux_amd64 && chmod +x $(LOCALBIN)/operator-sdk
```

Comment:
I think this is missing some `( )`.  Without them, if `test -s` passes, then it tries to `chmod +x $(LOCALBIN)/operator-sdk`, which may not exist.  This is the error I see:

```
chmod: cannot access '/home/ocp/podified/yazug-openstack-baremetal-operator/bin/operator-sdk': No such file or directory
make: *** [Makefile:354: /home/ocp/bin/operator-sdk] Error 1
```

Reply:
...so think we need to put `( )` around:

```
curl -o $(LOCALBIN)/operator-sdk -L https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_linux_amd64 && chmod +x $(LOCALBIN)/operator-sdk
```

Reply:
yea fixed, that was a duplicate